src/atoms/Switch/Switch.stories.js

```diff
@@ -0,0 +1,63 @@
+/*eslint-disable no-alert*/
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withKnobs } from '@storybook/addon-ondevice-knobs';
+import { select } from '@storybook/addon-knobs';
+import { View } from 'react-native';
+
+import Switch from './Switch';
+
+const alignOptions = {
+  left: 'left',
+  center: 'center',
+  right: 'right',
+};
+
+const sizeOptions = {
+  medium: 'medium',
+  large: 'large',
+};
+
+storiesOf('Toggle', module)
+  .addParameters({
+    component: Switch,
+  })
+  .addDecorator(withKnobs)
+  .add('Switch', () => (
+    <View style={{ height: '100%', justifyContent: 'space-around', alignItems: 'center' }}>
```

Comment:
Probably don't need to do this. Rebase with master, remove this and see how it works